Src/AutoNSubstitute.sln

```diff
@@ -11,6 +11,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoNSubstituteUnitTest", "
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoNSubstitute", "AutoNSubstitute\AutoNSubstitute.csproj", "{189C9474-B406-4356-A367-FBC882E9BA4B}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AutoFixture.xUnit.net", "AutoFixture.xUnit.net\AutoFixture.xUnit.net.csproj", "{266B9F0E-28E5-47CA-9816-ACB6737E7E4D}"
```

Comment:
I see.

These integration tests do serve a purpose (at least because they validate that what you want do to actually works).

What if someone comes by and wants to add a similar set of integration tests for *AutoFixture.NUnit2?* Or what if we get a new unit testing framework glue library? Should we add references to NUnit etc. to this solution as well?

It reasonable to have these integration tests, although we've traditionally haven't had them; IIRC, I don't think we have any integration tests that demonstrate the combination of xUnit.net and Moq.

Still, if we want to keep these integration tests, we'll need to create a new solution for them, as I don't want the coupling between xUnit.net and NSubstitute in the *AutoNSubstitute* solution.